packages/amplication-server/src/core/workspace/workspace.service.ts

```diff
@@ -789,4 +915,30 @@ export class WorkspaceService {
     await Promise.allSettled(promises);
     return hasChanges;
   }
+
+  async createResourceCustomActionsFix(
+    resources: Resource[],
+    user: User
+  ): Promise<boolean> {
+    let hasChanges = false;
+    const promises = resources.map(async (resource) => {
+      try {
+        for (const entity of resource.entities) {
+          hasChanges = await this.createEntityCustomActionsFix(
```

Comment:
If one entity has changes and the next doesn't, this will be false eventually. I think you might mean something like:
hasChanges = await this.createEntityCustomActionsFix(...) || hasChanges

Reply:
Done @mulygottlieb